bodyshopbidsdotcom/git-scripts
# Activate Virtualenv On Run

According to the setup instructions, we should be using virtualenvwrapper with git-scripts. However, the current implementation assumes it will be active when running.

This PR activates the environment every time a git-script is run. This ensures there won't be any library conflicts. This is especially important with the AWS libraries, where versions of awscli installed through Homebrew can differ from those included via pip.

This will require some users to re-run `git_setup`.

# Testing
1. Checkout this branch.
1. Run `git_setup`.
1. Verify your git-script functions still work: `git_run_eb ...`, `git_branch_from_task`, etc
